test/jpypetest/test_coverage.py

```diff
@@ -87,22 +86,27 @@ def testJArraySetItemSlice(self):
         # FIXME ja[0:-1:2] = [-1] works but should not
         # FIXME ja[0:-1:2] = 1 gives wierd error
 
-    def testJArrayGetItemSlice(self):
+    def testJArrayGetItemSlice_2(self):
         ja = jpype.JArray(jpype.JInt)([1, 2, 3, 4])
-        ja[1:]
+        assert list(ja[1:]) == [2, 3, 4]
 
-    def testJArraySetItemSlice(self):
+    def testJArraySetItemSlice_2(self):
         ja = jpype.JArray(jpype.JInt)([1, 2, 3, 4])
         ja[1:] = [3, 4, 5]
         self.assertEqual(list(ja[:]), [1, 3, 4, 5])
 
     def testJArrayEQ(self):
-        ja = jpype.JArray(jpype.JInt)([1, 2, 3, 4])
-        ja == [1, 2]
+        ja1 = jpype.JArray(jpype.JInt)([1, 2, 3, 4])
+        ja2 = jpype.JArray(jpype.JInt)([1, 2, 3, 4])
+        ja1 == ja2
+        # FIXME: This test is failing, but shouldn't be.
+        # assert ja1 == ja2
```

Comment:
@Thrameos - since I have your attention, do you happen to know why this test is failing? Is it related to your equality work in #1121?

I don't want to solve it in _this_ PR though :wink:

Reply:
Seems unlikely as the other was to do with comparable and not arrays which do not have java Comparable contracts.  Either way leave it to me to fix after this is merged.